client/dashboard/app/dataviews/dataviews.tsx

```diff
@@ -10,19 +10,32 @@ export type DataViewsProps< Item > = WPDataViewsProps< Item > & {
 	onResetView?: () => void;
 };
 
-export function DataViews< Item >( { view, onResetView, ...props }: DataViewsProps< Item > ) {
+export function DataViews< Item >( {
+	view,
+	onResetView,
+	header,
+	...props
+}: DataViewsProps< Item > ) {
 	const sanitizedView = sanitizeView( view, props.fields );
 
 	// TODO: apply local styles if necessary.
 	return (
 		<WPDataViews< Item >
 			view={ sanitizedView }
 			header={
-				onResetView && (
-					<Button variant="tertiary" size="compact" style={ { order: -1 } } onClick={ onResetView }>
-						{ __( 'Reset view' ) }
-					</Button>
-				)
+				<>
+					{ header }
```

Comment:
Pinging @Automattic/lego since we want to be very specific on the goals of this local DataViews component. Adding a `header` seems a bit generic, perhaps we could restrict it to a list of additional buttons?

Reply:
Ah, this is actually a correct change. `header` is an actual DataViews prop; we're just appending the Reset view button to it. I already make similar change [here](https://github.com/Automattic/wp-calypso/pull/107301/files#diff-88d9d901ca8325ed4a6cda012b9cb27c827422d32da42510888efa25059b24d3R16) (just merged).